x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/sync_jobs/documents_panel.tsx

```diff
@@ -45,7 +45,7 @@ export const SyncJobDocumentsPanel: React.FC<SyncJobDocumentsPanelProps> = (sync
       name: i18n.translate('xpack.enterpriseSearch.content.index.syncJobs.documents.volume', {
         defaultMessage: 'Volume',
       }),
-      render: (volume: number) => new ByteSizeValue(volume).toString(),
+      render: (volume: number) => new ByteSizeValue(volume * 1024 * 1024).toString(),
```

Comment:
Can we render as MB instead of bytes? This logic will lead to significant rounding errors, since we're storing integers, not floats. Also, I think it's likely that `volume` will frequently be `0`, in which case I think we should render "less than 1 MB" instead of "0", so that the user doesn't think that their was no data indexed.

Reply:
Sander and I ended up continuing this thread [here](https://elastic.slack.com/archives/C01795T48LQ/p1681889846003139).

> This logic will lead to significant rounding errors, since we're storing integers, not floats

I may have misunderstood this ByteSizeValue class at first - I though it was only for "byte size" and not "MB". So if we had `2000000` bytes -> `indexed_document_volume: 1` -> `new ByteSizeValue(1048576)`, that seemed like some bad rounding behavior. But now I understand that this will not display the large bytes number, but the smaller MB number. Still rounded, but not so terrible. I still think we should be more aproximate in our display, see [Sander's suggestion](https://elastic.slack.com/archives/C01795T48LQ/p1681921286839449?thread_ts=1681889846.003139&cid=C01795T48LQ). 

> We couldn't tell whether this is less than 1 MB or really 0. It's possible that a sync job syncs nothing (e.g. there's no change since last sync).

This is true. But a true `0` is still "less than 1MB", so I wouldn't be worried about that being misleading. Especially because this metric will be paired with a document count.